### ISSUE TITLE
React Component and WCAG 2.2 Level AA standards

### ISSUE THREAD

**Issue by jamiegood** [knocklabs/javascript#182]
https://www.equalweb.com/11312/11528/Web_Accessibility_Checker
latest WCAG 2.2 Level AA standards

Hi there, 

just spotted some accessibiliy issues on the knocklabs react library. 

This is important also considering the European Accessibility Act 2025.
https://ec.europa.eu/social/main.jsp?catId=1202&intPageId=5581&langId=en

See 3 issues flagged with the knock react library. I am using this tool
https://www.equalweb.com/11312/11528/Web_Accessibility_Checker
Checks for the latest WCAG 2.2 Level AA standards. Plus a description of the issue below.

It's spotting these issues in the Knock React feed. 
The first 2 seem related to the Select tag on the notifications feed. 

SC 1.3.1: INFO AND RELATIONSHIPS (LEVEL A)
[id="root"] > header > div > div > div:nth-child(3) > div > div > div > div > header > div > div > select
This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.

SC 4.1.2: NAME, ROLE, VALUE (LEVEL A)
[id="root"] > header > div > div > div:nth-child(3) > div > div > div > div > header > div > div > select
This select element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.

SC 4.1.2: NAME, ROLE, VALUE (LEVEL A)
[id="root"] > header > div > div > div:nth-child(3) > div > div > div > div > div:nth-child(3) > a
Anchor element found with a valid href attribute, but no link content has been supplied.

Thought I'd raise this issue. 

Thanks
Jamie
